### PR TITLE
Auto commands

### PR DESCRIPTION
**Pull request by jnallard**
Creating a pull request for this early so we can add some comments.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/SwerveModule.java [hunk 1]
```diff
@@ -53,6 +53,11 @@ public void setTarget(SwerveModuleState state) {
     m_targetState = state;
   }
 
+  public void stop() {
+    m_angle.set(TalonFXControlMode.PercentOutput, 0);
```
Comment (jnallard) [inline src/main/java/frc/robot/subsystems/SwerveModule.java:58]: We need to tell the simulator to stop moving too

File: src/main/java/frc/robot/subsystems/SwerveDrive.java [hunk 2]
```diff
@@ -142,6 +142,16 @@ public void moveRobotRelative(double xForwardSpeedMetersPerSecond, double ySidew
     move(speeds);
   }
 
+  public void resetPids() {
+    m_XPid.reset();
+    m_YPid.reset();
+    m_AnglePid.reset();
```
Comment (jnallard) [inline src/main/java/frc/robot/subsystems/SwerveDrive.java:149]: we need to make the PID continuous

File: src/main/java/frc/robot/subsystems/SwerveDrive.java [hunk 3]
```diff
@@ -142,6 +142,16 @@ public void moveRobotRelative(double xForwardSpeedMetersPerSecond, double ySidew
     move(speeds);
   }
 
+  public void resetPids() {
```
Comment (jnallard) [inline src/main/java/frc/robot/subsystems/SwerveDrive.java:146]: This didn't fix the issue when running the auto twice

Sample auto:
```
resetPosition?x=8&y=4
driveToTarget?x=12&y=6&angle=90

```

**Review by huyha23 — APPROVED** [review]

**Review by aqhoang2005 — APPROVED** [review]

**Review by NinjaBuilderPro — APPROVED** [review]
Approve as a group

**Pull request merged**